do_mpc/controller.py

```diff
@@ -1150,6 +1155,36 @@ def _prepare_nlp(self):
             entry('_u', repeat=[self.n_horizon, n_u_scenarios], struct=self.model._u),
             entry('_eps', repeat=[n_eps, n_max_scenarios], struct=self._eps),
         ])
+        """
+        # Build discrete vector for MINLP
```

Comment:
Why is there a long comment section here? Please remove this if it is not relevant.